src/main/java/io/github/tryexceptelse/jdex/be/Contact.java

```diff
@@ -20,57 +25,95 @@ public void Contact(String newFirst, String newLast, String newEmail, String new
         notes=newNotes;
     }
     /*
-     * the following methods can be called to set set the variable of the same name
+     *  can be called to set set first ME
      */
     public void setFirst(String newFirst)
```

Comment:
and all of these setters and getters can be replaced with getEntry(String Entry) and setEntry(String Entry, String newValue)